### PR TITLE
[Little Simz] improve Venom

### PR DESCRIPTION
**Pull request by egecelikci**
https://github.com/Lyrics/lyrics/pull/319 (I'm sorry I'm a little late.)

### CODE REVIEW

File: database/L/Little Simz/GREY Area/Venom [hunk 1]
```diff
@@ -1,87 +1,79 @@
...
+So think twice if you wanna get blazed
+Nothin' ain't nice 'round here, no games
+I don't wanna dark nobody in a flash
+True, it's a stick-up, hands in the air
+Put the money in the bag, all cash, all cash
 
 Venom
 Venom
+Venom
+Venom, venom
```
Comment (snshn) [inline database/L/Little Simz/GREY Area/Venom]: In the youtube video version "Venom" here is read only once, don't have the album/song so I may be using wrong reference

File: database/L/Little Simz/GREY Area/Venom [hunk 2]
```diff
@@ -1,87 +1,79 @@
...
+
 Venom
```
Comment (snshn) [inline database/L/Little Simz/GREY Area/Venom]: In the youtube video this line is absent, but if it's present on the album version I believe it'd be better to keep it.

File: database/L/Little Simz/GREY Area/Venom [hunk 3]
```diff
@@ -1,87 +1,79 @@
...
+Niggas pussy lickin', batty and shit
+Oh, you mad? Then come at me, you prick
+Make a move, better pattern it quick
+I assume you'll be comin' for blood
+That makes two of us, that makes two of us
 
 Venom
 Venom
 Venom
+Venom, venom
```
Comment (snshn) [inline database/L/Little Simz/GREY Area/Venom]: I believe there's a slight echo of "venom" after the second time on this line, could be added as `(venom, venom)`

File: database/L/Little Simz/GREY Area/Venom (hunk 2 above)
Comment (egecelikci) [reply database/L/Little Simz/GREY Area/Venom]: Yes, I've misplaced it I think. It's same on the album version too. Fixed on new commit

File: database/L/Little Simz/GREY Area/Venom (hunk 1 above)
Comment (egecelikci) [reply database/L/Little Simz/GREY Area/Venom]: No, she says it three times on both (album & YouTube) versions. (You can check it again if you want 1:12, 1:15, 1:18)

File: database/L/Little Simz/GREY Area/Venom (hunk 3 above)
Comment (egecelikci) [reply database/L/Little Simz/GREY Area/Venom]: I think you're right, I've changed 43rd and 75th lines as `(venom, venom)`.

**Pull request merged**